deploy/vagrant/core/config_aws.rb

```diff
@@ -20,7 +22,7 @@ def config_aws(config, i, total, name)
     aws.instance_type = INSTANCE_TYPE
     aws.block_device_mapping = BLOCK_DEVICE_MAPPING
     aws.tags = {
-      'Name' => TAG + name,
+      'Name' => TAG + "-" + name,
```

Comment:
Does this use the tag from the config file? In the comments in the config file, it says that it will be {Tag}TachyonMaster. If the script will be adding the '-', then that should be reflected in the comments and example in ec2.yml.